apis/apps/v1alpha1/component_types.go

```diff
@@ -82,6 +92,11 @@ type ComponentSpec struct {
 	// +patchStrategy=merge,retainKeys
 	VolumeClaimTemplates []ClusterComponentVolumeClaimTemplate `json:"volumeClaimTemplates,omitempty" patchStrategy:"merge,retainKeys" patchMergeKey:"name"`
 
+	// List of volumes to override.
+	//
+	// +optional
+	Volumes []corev1.Volume `json:"volumes,omitempty"`
```

Comment:
Why is this needed? How is this related to https://github.com/apecloud/kubeblocks/issues/5829 for enabling Multus CNI ?

Reply:
Both of them are about cluster annotations.

Reply:
https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#launch-pod-with-text-annotation